src/enums/name-request-codes.ts

```diff
@@ -5,14 +5,6 @@
 
 // ref: https://github.com/bcgov/namex/blob/main/api/namex/constants/__init__.py
 export enum NrRequestTypeCodes {
-  // DEPRECATION WARNING - these will be removed!
-  BC_COMPANY = 'BC', // use NEW_BC instead
-  BC_CORPORATION = 'CR', // use NEW_CORP instead
-  BC_UNLIMITED = 'UL', // use NEW_ULC instead
-  COOP = 'CP', // use NEW_COOP instead
-  NR_SOLE_PROP = 'FR', // use NEW_FIRM instead
-  XPRO_CORPORATION = 'XCR', // use NEW_XPRO_CORP instead
```

Comment:
I couldn't find these used in Filings UI/Create UI/Edit UI/Auth Web/Namerequest UI (and there are alternate enums below with these values) so I deleted these.